pyproject.toml

```diff
@@ -26,15 +71,45 @@ extend-select = [
     "UP",
 ]
 ignore = [
```

Comment:
Please also add a `select = ["ALL"]`

Reply:
done in 7d409fa + fix all the problems 🎉